infra/Makefile

```
@@ -0,0 +1,127 @@
+.PHONY: help build run test test-coverage fmt lint migrate-up migrate-down migrate-status docker-up docker-down clean test-local docker-prod-up generate-secrets create-env test-db fix-mysql-permissions setup-env-files
+
+# Default target
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (18).

(maxbodylength)